specification/components/stu3/schemas/DocumentReference.yaml

```diff
@@ -114,15 +114,15 @@ properties:
               example: '2021-04-22'
             extension:
               type: array
-              minItems: 3
+              minItems: 2
               maxItems: 4
               description: |
                 
                 | Field                                   | Cardinality | Notes                                      |
                 | --------------------------------------- | ----------- | ------------------------------------------ |
                 | `eRS-AvailabilityStatus`                | `1..1`      |                                            |
                 | `eRS-AttachedBy`                        | `1..1`      |                                            |
-                | `eRS-AttachedByOrganisation`            | `1..1`      |                                            |
+                | `eRS-AttachedByOrganisation`            | `0..1`      |                                            |
```

Comment:
Looks good. Is it worth putting a comment in the Notes column? If not, could we consider dropping it? Also, after we fix the issue, this will go back to `1..1` I think - given we were considering null constraints and all. Just wondering if it's worth raising something for it, or add to the ticket's scope if one already exists, so we don't forget about it?

Reply:
Fair point, I updated ERSSUP-82540 to raise the OAS change in case we can revert the change. About the Notes, lets discussed, I agree at the moment doesn't add much value